libcore/gof-file.c

```diff
@@ -1366,13 +1366,13 @@ gof_file_is_executable (GOFFile *file)
     {
         /* get the content type of the file */
         content_type = gof_file_get_ftype (file);
+
         if (G_LIKELY (content_type != NULL))
         {
             /* check if the content type is save to execute, we don't use
              * g_content_type_can_be_executable() for unix because it also returns
              * true for "text/plain" and we don't want that */
-            if (g_content_type_is_a (content_type, "application/x-executable")
-                || g_content_type_is_a (content_type, "application/x-shellscript"))
+            if (g_content_type_is_a (content_type, "application/x-executable"))
                 can_execute = TRUE;
```

Comment:
By convention, we convert lines touched to elementaryos style even if rest of file does not comply.

Please use braces with single line if clause.

```
if () {

}
```

Reply:
I'm a newcomer. This is me second pull request. Thanks you for comments!